config/routes.rb

```diff
@@ -8,6 +8,7 @@
   end
   get "static_pages/home"
 
+  resources :actor_categories, except: [:edit, :update]
```

Comment:
@parndt curious about the `except` bit here that appears to be inconsistent with other, similar routes (eg `measure_categories`)

Reply:
`resources` defines 7 routes: `index`, `show`, `edit`, `update`, `new`, `create`, `destroy` so here we are saying we want them all `except: `[:edit, :update]`

From the [Rails routing guide](https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions):
<img width="637" alt="image" src="https://user-images.githubusercontent.com/10128/140813804-030168a4-11c1-47bf-b86d-67fb7bae9dca.png">

This is the same as saying

```suggestion
  resources :actor_categories, only: [:index, :show, :new, :create, :destroy]
```

Happy to switch it to `only` as it matches the others

Reply:
thanks for the details and update! happy either way =)